Python/pystate.c

```diff
@@ -665,6 +665,8 @@ init_interpreter(PyInterpreterState *interp,
         /* Fix the self-referential, statically initialized fields. */
         interp->dtoa = (struct _dtoa_state)_dtoa_state_INIT(interp);
     }
+    llist_init(&interp->threads.daemon_handles);
+    llist_init(&interp->threads.non_daemon_handles);
```

Comment:
It would make sense to have a `_PyThread_InitThreadHandles()` for this, to match `_PyThread_ClearThreadHandles()`.